app_dart/lib/src/request_handlers/update_agent_health_history.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:collection/collection.dart';
+import 'package:gcloud/db.dart';
+import 'package:googleapis/bigquery/v2.dart';
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+import '../model/appengine/agent.dart';
+import '../request_handling/api_request_handler.dart';
+import '../request_handling/authentication.dart';
+import '../request_handling/body.dart';
+import '../service/datastore.dart';
+
+const Duration maxHealthCheckAge = Duration(minutes: 10);
+
+@immutable
+class UpdateAgentHealthHistory extends ApiRequestHandler<Body> {
+  const UpdateAgentHealthHistory(
+    Config config,
+    AuthenticationProvider authenticationProvider, {
+    @visibleForTesting
+        this.datastoreProvider = DatastoreService.defaultProvider,
+  }) : super(config: config, authenticationProvider: authenticationProvider);
+
+  final DatastoreServiceProvider datastoreProvider;
+
+  @override
+  Future<Body> get() async {
+    const String projectId = 'flutter-dashboard';
```

Comment:
Should these be class members instead so they include documentation on what they are for?

Reply:
I used to put those as class members, but got comments to include them in specific sub functions to make them more containable. I agreed to change them accordingly. After all, these variables only make sense for this sub function, considering if we have more other sub functions in the future.

Reply:
I still think they need some documentation. Project id refers to the Google Cloud project that is currently running. `dataset` and `table` should be documented as variables for BigQuery.

Reply:
Added one comment: 
// Define const variables for [BigQuery] operations.